tools/ansible/tasks/wsu/main.yaml

```diff
@@ -99,3 +104,25 @@
       delegate_to: localhost
       shell: "oc adm certificate approve {{ item }}"
       with_items: "{{ node_csrs.stdout_lines }}"
+
+    - name: Get bootstrapped node name
+      delegate_to: localhost
+      shell: "oc get no -o wide |awk '/{{ inventory_hostname }}/ {print $1}'"
```

Comment:
Nit: Using `oc get nodes` makes context clearer compared to `oc get no`
Nit: Maybe a comment about `{{ inventory_hostname }}` pointing to node ip provided in hostfile?